core/src/banking_stage/transaction_scheduler/scheduler_controller.rs

```diff
@@ -742,7 +755,7 @@ mod tests {
             bank_forks,
             PrioGraphScheduler::new(consume_work_senders, finished_consume_work_receiver),
             vec![], // no actual workers with metrics to report, this can be empty
-            forwarder,
+            None,
```

Comment:
Nah these tests explicitly call into the scheduler functions instead of running the main loop.
None of them call into actual forwarding functions right now.